### DISCUSSION TITLE
Schedule on Right?  Onyx Note Air Version?

### DISCUSSION THREAD

**Discussion by badramrod** [kudrykv/latex-yearly-planner#26] [Ideas]
Any chance you will release a version that switches the columns in the daily view so that the schedule/calendar is on the right and the priorities and notes are on the left?

Also, any chance an Onyx version will be released?  I think the only change needed from the RM2 versions is to shift the top right link (e.g., the next day in the breadcrumb view) about an inch to the left.  Otherwise the bookmark icon in the Onyx reader interferes with the link.

Thanks!

**Comment by kudrykv** [comment]
> Any chance you will release a version that switches the columns in the daily view so that the schedule/calendar is on the right and the priorities and notes are on the left?

The problem with variations is that they exponentially increase the number of PDFs. Today, I get about 64 variations for a single year for a template. So two templates + two years result in 256 compiled PDFs.

Still, it makes sense to have what you're asking if that is useful. The thing is to figure out a convenient way of adding new variations and create an executable that would help build those.

Saying that it should be easy enough to assemble a single PDF, as it requires mostly only configuration changes.

> Also, any chance an Onyx version will be released?

Do you own an Onyx boox? It would be great if you could measure the exact screen size in mm, and also open any PDF and measure the size of all control buttons it has.

**Comment by badramrod** [comment]
Thanks for the quick reply. Regarding the variations, that makes sense. You
already have a ton of variations here.

Regarding the Onyx, yes I have a Boox Note Air 2. I am using

Rm2_ddvk.breadcrumb.lines.default.ampm.sun.daily cal.2022. The bookmark
button (static position) is in the upper right hand corner and appears to
extend 1/4 of an inch (~ 7 mm) to the left. The floating toolbar with cam
be oriented vertically or horizontally, but can’t be dragged all the way to
the top. So it is best if oriented vertically and positioned on the left
hand side for a right hand writer.  It looks to extend out from the side
3/8 of an inch (~10 mm).  Pictures attached.

Also, total screen dimensions appear to be 8.25 inches x 6.125 (sorry, don’t have a metric L ruler).

This may be a good time to ask - what exactly is the difference between the
rm2 versions and the ddvk versions?

Thanks


On Mon, Feb 7, 2022 at 3:54 AM Vitaliy Kudryk ***@***.***>
wrote:

> Any chance you will release a version that switches the columns in the
> daily view so that the schedule/calendar is on the right and the priorities
> and notes are on the left?
>
> The problem with variations is that they exponentially increase the number
> of PDFs. Today, I get about 64 variations for a single year for a template.
> So two templates + two years result in 256 compiled PDFs.
>
> Still, it makes sense to have what you're asking if that is useful. The
> thing is to figure out a convenient way of adding new variations and create
> an executable that would help build those.
>
> Saying that it should be easy enough to assemble a single PDF, as it
> requires mostly only configuration changes.
>
> Also, any chance an Onyx version will be released?
>
> Do you own an Onyx boox? It would be great if you could measure the exact
> screen size in mm, and also open any PDF and measure the size of all
> control buttons it has.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kudrykv/latex-yearly-planner/discussions/26#discussioncomment-2123997>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXUOHY2DHEN7FTLE5GEAN5DUZ6JGHANCNFSM5NV65DWA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Sent from my mobile

**Reply by kudrykv** [reply]
> what exactly is the difference between the rm2 versions and the ddvk versions?

Rm2 has an empty space in the upper right corner, ddvk doesn't. The size actually close if not the same with the Remarkable devices, so it might fit your needs — [rm2.breadcrumb.lined.default.ampm.sun.dailycal.2022](https://github.com/kudrykv/latex-yearly-planner/releases/download/v0.0.8/rm2.breadcrumb.lined.default.ampm.sun.dailycal.2022.pdf). The empty space at the left is still a little bit too much, but the top right corner is free.

**Reply by badramrod** [reply]
OK great - I'll give that one shot. Thanks!
